packages/app/app/containers/SoundContainer/index.js

```diff
@@ -130,6 +137,14 @@ class SoundContainer extends React.Component {
     });
   }
 
+  handleError() {
```

Comment:
hacky, maybe it'd be better to store a fallback/default plugin in redux (like we store the selected one) and use it when errors occur?

Reply:
sure, one more select in the plugins view ? or something hard coded ?

Reply:
Can be hardcoded somewhere, I just don't like depending on comparing string names